### ISSUE TITLE
もしもスマブラだったら: Actor (Picture) position and zoom incorrect

### ISSUE THREAD

**Issue by Ghabry** [EasyRPG/Player#1662]
This is a new issue caused by cd4fff9 but based on tests the behaviour in this commit is correct.

The real problem must be somewhere else.
Simply start a battle, you will see the problem - the actors are completely off and not zoomed.

#### Name of the game: もしもスマブラだったら (https://rmarchiv.tk/games/1412)

Web player: https://easyrpg.org/play/master/?game=smashbros

**Comment by fmatthew5876** [comment]
From Cherry's comment in another thread. Here is more about ordering:

```
Main update methods for common events run (they just check if they are parallel process and the condition is met and then invoke the parallel process worker for them)
Main update methods for map events run (they do a bunch of things, described in my previous post - but actually even more, because after the check for whether there is any active event page, they will run their parallel process worker and then check once more for active)
Message window updates
Timer updates
Screen effects update
Pictures update
Foreground worker runs
Panorama updates
```

May be related to this

**Comment by fmatthew5876** [comment]
Screen Effects Test Cases 1
--------------------------------

1. Setup EV01 as parallel
```
ShakeScreen Once, 5,,5 @1.0s
OpenSaveMenu
Erase Event
```

| Save | shake str | shake speed | shake pos | shake time left |
| -- | -- | -- | -- | -- |
| 1 | 5 | 5 | 0 | 60 |

Result: screen effects don't happen during pre-update

2. Setup EV01 as parallel
```
Wait 0.0s
ShakeScreen Once, 5,,5 @1.0s
OpenSaveMenu
Erase Event
```

| Save | shake str | shake speed | shake pos | shake time left |
| -- | -- | -- | -- | -- |
| 1 | 5 | 5 | 3 | 59 |

Result: screen effect animates

3. Setup EV01 as autostart
```
ShakeScreen Once, 5,,5 @1.0s
OpenSaveMenu
Erase Event
```

| Save | shake str | shake speed | shake pos | shake time left |
| -- | -- | -- | -- | -- |
| 1 | 5 | 5 | 0 | 60 |

Result: Screen effect doesn't animate - happens before auto start event

**Comment by fmatthew5876** [comment]
Fixed by #1701
